src/encoded/schemas/experiment_series.json

```diff
@@ -0,0 +1,211 @@
+                {
+                    "not": {
+                        "properties": {
+                            "status": {
+                                "enum": ["released", "revoked"]
+                            }
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "accession": {
+            "accessionType": "SR"
+        },
+        "award": {
+            "default": "5854a1dc-4c50-44b3-ba58-0a6f4f399297",
+            "linkEnum": [
+                "5854a1dc-4c50-44b3-ba58-0a6f4f399297"
+            ],
+            "permission": "import_items"
+        },
+        "date_released": {
+            "title": "Date released",
+            "comment": "Do not submit, value is assigned whe the object is releaesd.",
+            "type": "string",
+            "anyOf": [
+                {"format": "date-time"},
+                {"format": "date"}
+            ],
+            "permission": "import_items"
+        },
+        "lab": {
+            "default": "cb0ef1f6-3bd3-4000-8636-1c5b9f7000dc",
+            "linkEnum": [
+                "cb0ef1f6-3bd3-4000-8636-1c5b9f7000dc"
+            ],
+            "permission": "import_items"
+        },
+        "related_datasets": {
+            "minItems": 1
+        },
+        "status": {
+            "title": "Status",
+            "type": "string",
+            "permission": "import_items",
+            "default": "in progress",
+            "enum" : [
+                "in progress",
+                "submitted",
+                "released",
+                "archived",
+                "deleted",
+                "replaced",
+                "revoked"
+            ],
+            "permission": "import_items"
+        }
+    },
+    "facets": {
+        "assay_slims": {
+            "title": "Assay type"
+        },
+        "assay_titles": {
+            "title": "Assay title"
+        },
+        "biosample_ontology.cell_slims": {
+            "title": "Cell"
+        },
+        "biosample_ontology.classification": {
+            "title": "Biosample classification"
+        },
+        "biosample_ontology.organ_slims": {
+            "title": "Organ"
+        },
+        "biosample_ontology.term_name": {
+            "title": "Biosample term name"
+        },
+        "contributing_awards.project": {
+            "title": "Project"
+        },
+        "contributing_awards.rfa": {
+            "title": "RFA"
+        },
+        "contributors.title": {
+            "title": "Lab"
+        },
+        "date_released": {
+            "title": "Month released"
+        },
+        "organism.scientific_name": {
+            "title": "Organism"
+        },
+        "status": {
+            "title": "Status"
+        },
+        "target.investigated_as": {
+            "title": "Target category"
+        },
+        "target.label": {
```

Comment:
Should be:
        "target.label": {
            "title": "Target of assay",
            "type": "typeahead",
            "length": "long"
        },